### ISSUE TITLE
More Options

### ISSUE THREAD

**Issue by gustavochavarria** [gustavochavarria/save-html-as-image#3]
Many options are not implemented for now

https://github.com/bubkoo/html-to-image#options

**Comment by MentalGear** [comment]
Yes, indeed, several options like the filter or cache busting option would be very helpful.
